src/playerOverview/PlayerScreen.tsx

```diff
@@ -208,17 +193,15 @@ export const PlayerScreen = React.memo(function PlayerScreen() {
                     alignItems: 'center',
                 }}
             >
-                <h1
-                    style={{
-                        paddingLeft: 8,
-                        fontSize: 32,
-                        fontWeight: 'bold',
-                        fontStyle: 'italic',
-                        alignSelf: 'flex-start',
-                    }}
+                <Heading

```

Comment:
maybe something to check with @nsthi , but i like the headings to be all caps-- just my 2 cents, happy to align to this style for all headers

Reply:
Mad everything caps! WE YELLIN NOW